File: branches/merge0706/code/contrib/nspatialdb/bin/stdlight.lua
-------------------------------------------------------------------------------
--   Defines the standard lighting for nviewer.exe
-------------------------------------------------------------------------------

pushcwd('.')

sel('/usr/scene')

new('ntransformnode','stdlight')
    sel('stdlight')
    new('nlightnode', 'l')
	l = lookup('l')
	sel('l')
	l:setposition(1000,500,1000)
	l:setvector('LightDiffuse',0.8,0.8,0.8,1)
	l:setvector('LightSpecular',1,1,1,1)
	l:setvector('LightAmbient',0.2,0.2,0.2,1)
	l:setvector('LightDiffuse1', 0.3, 0.3, 0.3, 1.0)
	l:setvector('LightSpecular1', 0.3, 0.3, 0.3, 1.0)
    sel('..') 
sel('..') 

popcwd()

File: nebula2/code/contrib/nrubyserver/bin/stdlight.rb
#-------------------------------------------------------------------------------
#   Defines the standard lighting for nviewer.exe
#-------------------------------------------------------------------------------

oldCwd = psel
sel '/usr/scene'

new 'ntransformnode', 'stdlight'
    sel 'stdlight'
        new 'nlightnode', 'l'
        sel 'l'
        setposition 1000, 500, 1000
        setvector 'LightDiffuse',   1, 1, 1, 1
        setvector 'LightSpecular',  1, 1, 1, 1
        setvector 'LightAmbient' ,  1, 1, 1, 1
        setvector 'LightDiffuse1' , 0.3, 0.3, 0.3, 1.0
        setvector 'LightSpecular1', 0.3, 0.3, 0.3, 1.0

sel oldCwd




